### PR TITLE
Create DB

### PR DESCRIPTION
**Pull request by Sviridy**

### CODE REVIEW

File: department/department_app/models/__init__.py [hunk 1]
```diff
@@ -0,0 +1,5 @@
+"""Models"""
```
Comment (inauros) [inline department/department_app/models/__init__.py:1]: Please use only absolute imports in your project. Relative imports can cause many unexpected problems in the future :)

File: department/department_app/models/department.py [hunk 2]
```diff
@@ -0,0 +1,13 @@
+"""models"""
+from django.db import models
+
+
+class Department(models.Model):
+    """Department"""
+    department = models.CharField(max_length=25)
+    dumber_of_employees = models.IntegerField(null=True)
+    the_average_salary = models.IntegerField(null=True)
```
Comment (inauros) [inline department/department_app/models/department.py]: In accordance with the task, the average salary must be calculated automatically. Please remove this field and don't forget to generate a new migration script.

File: department/department_app/models/department.py [hunk 3]
```diff
@@ -0,0 +1,13 @@
+"""models"""
+from django.db import models
+
+
+class Department(models.Model):
+    """Department"""
+    department = models.CharField(max_length=25)
+    dumber_of_employees = models.IntegerField(null=True)
```
Comment (inauros) [inline department/department_app/models/department.py]: **D**umber_of_employees? :) I would recommend removing this field and generate it automatically (as for average_salary).

**Review by inauros — CHANGES_REQUESTED** [review]
1. Please remove the db.sqlite3 file
2. In accordance with the task, it was necessary not only to create a database but also to fill it with test data. Create an additional migration script to fill the database with test data.

File: department/department_app/models/__init__.py (hunk 1 above)
Comment (Sviridy) [reply department/department_app/models/__init__.py:1]: fixed

File: department/department_app/models/department.py (hunk 2 above)
Comment (Sviridy) [reply department/department_app/models/department.py]: fixed

File: department/department_app/models/department.py (hunk 3 above)
Comment (Sviridy) [reply department/department_app/models/department.py]: fixed

File: department/department_app/migrations/0002_auto_20210825_1231.py [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+    Department.objects.create(department="Developer")
+    Specialization.objects.create(name='Python')
+    Specialization.objects.create(name='C#')
+    Specialization.objects.create(name='Java')
+    Specialization.objects.create(name='C++')
+    Positions.objects.create(name='Senior')
+    Positions.objects.create(name='Middle')
+    Positions.objects.create(name='Junior')
+    Positions.objects.create(name='Student')
+
```
Comment (inauros) [inline department/department_app/migrations/0002_auto_20210825_1231.py]: please add a function to revert db updates

File: department/department_app/migrations/0002_auto_20210825_1231.py [hunk 5]
```diff
@@ -0,0 +1,30 @@
...
+    Department.objects.create(department="Marketing")
+    Department.objects.create(department="Developer")
+    Specialization.objects.create(name='Python')
+    Specialization.objects.create(name='C#')
+    Specialization.objects.create(name='Java')
+    Specialization.objects.create(name='C++')
+    Positions.objects.create(name='Senior')
+    Positions.objects.create(name='Middle')
+    Positions.objects.create(name='Junior')
+    Positions.objects.create(name='Student')
```
Comment (inauros) [inline department/department_app/migrations/0002_auto_20210825_1231.py]: please, use bulk_insert

File: department/department_app/migrations/0003_auto_20210825_1250.py [hunk 6]
```diff
@@ -0,0 +1,19 @@
+# Generated by Django 3.2.6 on 2021-08-25 09:50
+
+from django.db import migrations
+
+def data_filling(apps, schema_editor):
+    Employee = apps.get_model("department_app", "Employee")
+    Employee.objects.create(name='Ihar Nauros', date_of_birth='01.01.1990', department_id=36, experience=15,
+                            wages=1500, positions_id=41, spesialization_id=41)
+    Employee.objects.create(name='Svirydzetski Kanstantsin', date_of_birth='05.05.1998', experience=1, wages=10,
+                            department_id = 36, positions_id = 44, spesialization_id = 41)
```
Comment (inauros) [inline department/department_app/migrations/0003_auto_20210825_1250.py]: please use bulk_insert

File: department/department_app/migrations/0003_auto_20210825_1250.py [hunk 7]
```diff
@@ -0,0 +1,19 @@
...
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('department_app', '0002_auto_20210825_1231'),
+    ]
+
+    operations = [
+    ]
```
Comment (inauros) [inline department/department_app/migrations/0003_auto_20210825_1250.py]: why operations is empty?

File: department/department_app/migrations/0003_auto_20210825_1250.py [hunk 8]
```diff
@@ -0,0 +1,19 @@
...
+
+from django.db import migrations
+
+def data_filling(apps, schema_editor):
+    Employee = apps.get_model("department_app", "Employee")
+    Employee.objects.create(name='Ihar Nauros', date_of_birth='01.01.1990', department_id=36, experience=15,
+                            wages=1500, positions_id=41, spesialization_id=41)
+    Employee.objects.create(name='Svirydzetski Kanstantsin', date_of_birth='05.05.1998', experience=1, wages=10,
+                            department_id = 36, positions_id = 44, spesialization_id = 41)
+
```
Comment (inauros) [inline department/department_app/migrations/0003_auto_20210825_1250.py]: please add a function to revert db updates

File: department/department_app/migrations/0004_auto_20210825_1254.py [hunk 9]
```diff
@@ -0,0 +1,21 @@
+# Generated by Django 3.2.6 on 2021-08-25 09:54
```
Comment (inauros) [inline department/department_app/migrations/0004_auto_20210825_1254.py]: this migration script looks like a duplicate

File: department/department_app/migrations/0005_auto_20210825_1258.py [hunk 10]
```diff
@@ -0,0 +1,21 @@
+# Generated by Django 3.2.6 on 2021-08-25 09:58
```
Comment (inauros) [inline department/department_app/migrations/0005_auto_20210825_1258.py]: this migration script looks like a duplicate

File: department/department_app/models/departmentt.py [hunk 11]
```diff
@@ -0,0 +1,27 @@
+"""models"""
+from django.db import models
+
```
Comment (inauros) [inline department/department_app/models/departmentt.py:3]: Please import Employee

File: department/department_app/migrations/0002_auto_20210825_1231.py (hunk 4 above)
Comment (Sviridy) [reply department/department_app/migrations/0002_auto_20210825_1231.py]: Fixed

File: department/department_app/migrations/0002_auto_20210825_1231.py (hunk 5 above)
Comment (Sviridy) [reply department/department_app/migrations/0002_auto_20210825_1231.py]: Fixed

File: department/department_app/migrations/0003_auto_20210825_1250.py (hunk 6 above)
Comment (Sviridy) [reply department/department_app/migrations/0003_auto_20210825_1250.py]: Fixed

File: department/department_app/migrations/0003_auto_20210825_1250.py (hunk 7 above)
Comment (Sviridy) [reply department/department_app/migrations/0003_auto_20210825_1250.py]: I made a mistake

File: department/department_app/migrations/0003_auto_20210825_1250.py (hunk 8 above)
Comment (Sviridy) [reply department/department_app/migrations/0003_auto_20210825_1250.py]: added

File: department/department_app/migrations/0004_auto_20210825_1254.py (hunk 9 above)
Comment (Sviridy) [reply department/department_app/migrations/0004_auto_20210825_1254.py]: Fixed

File: department/department_app/migrations/0005_auto_20210825_1258.py (hunk 10 above)
Comment (Sviridy) [reply department/department_app/migrations/0005_auto_20210825_1258.py]: Fixed

File: department/department_app/models/departmentt.py (hunk 11 above)
Comment (Sviridy) [reply department/department_app/models/departmentt.py:3]: Fixed

**Pull request merged**